src/components/ScreenWrapper/index.android.js

```diff
@@ -6,13 +6,12 @@ const ScreenWrapper = props => (
     <BaseScreenWrapper
         // eslint-disable-next-line react/jsx-props-no-spreading
         {...props}
+        keyboardAvoidingViewBehavior="height"
     >
         {props.children}
     </BaseScreenWrapper>
 );
 
-defaultProps.keyboardAvoidingViewBehavior = 'height';
-
 ScreenWrapper.propTypes = propTypes;
```

Comment:
I am simple reverting [this PR](https://github.com/Expensify/App/pull/10567/files)